snappy-tools/sbin/collect-debug-artifacts.sh

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env bash
+
+#
+# Copyright (c) 2016 SnappyData, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you
+# may not use this file except in compliance with the License. You
+# may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License. See accompanying
+# LICENSE file.
+#
+
+#!/usr/bin/env bash
+
+usage="Usage: collect-debug-artifacts [ -c|--conf|--config conffile ] [ -h|--help ] \
+       [ -a|--all ] [ -v|--verbose ]"
+
+PWD=`pwd`
+while [ "$1" != "" ]; do
+key="$1"
+
+case $key in
+  -c|--conf|--config)
+  CONF_FILE="$2"
```

Comment:
usual convention with double hyphen "--" is to have the value with an '=' rather than in next argument. So I will suggest like:

```sh
case $key in
  -c)
    CONF_FILE="$2"
    shift ;;
  --conf=*|--config=*)
    CONF_FILE="`echo "$2" | sed 's/^[^=]*=//'`" ;;
```